lib/components/BuilderUI.jsx

```diff
@@ -40,6 +41,13 @@ const sendPrizeStrategyTx = async (params, walletContext, chainId, setTx, setRes
     signer
   )
 
+  const singleRandomWinnerBuilderAddress = CONTRACT_ADDRESSES[chainId]['SINGLE_RANDOM_WINNER_BUILDER']
```

Comment:
So I just published a new library yesterday that let's us put these constants all in one place, ideally for each version of the contracts (ie. `alpha.34` would be a new NPM published version of the `@pooltogether/current-pool-data` repo)
https://github.com/pooltogether/current-pool-data

We should make use of that for any constant addresses, instead of hard-coding them manually in each repo.

Reply:
I don't know if this applies to builders, you can probably ignore it for this repo!